content/en/docs/releases/kubeflow-1.10.md

```diff
@@ -4,7 +4,7 @@ description = "Information about the Kubeflow 1.10 release"
 weight = 95
 +++
 
-## Kubeflow 1.10.0
+## Kubeflow 1.10.1
```

Comment:
Like previous releases which had patch releases, please dont delete the origional release, just add a new heading for it (in this case `1.10.1`).

See https://www.kubeflow.org/docs/releases/kubeflow-1.8/ for example.